clipper_admin/clipper_admin/sample_config.yaml

```diff
@@ -0,0 +1,42 @@
+models:
+  Model1:
+    input_type: "integers"
+    version: "1.2.7"
+    image: "test-model:1"
+  Model2:
+    input_type: "floats"
+    version: "1.2.7"
+    image: "train-model:2"
+    labels: ["second model configured", "working successfully"]
+    num_replicas: 2
+  Model3:
+    input_type: "doubles"
+    version: "4.8.9"
+    image: "test-model:2"
+    num_replicas: 3
+    batch_size: 2
+
+applications:
+  Application1:
+    obj_type: "application"
+    input_type: "doubles"
+    slo: 10000
+    default_value: "not received"
+  Application2:
+    obj_type: "application"
+    input_type: "bytes"
+    slo: 5000
+    default_value: "undetected"
+  Application3:
+    obj_type: "application"
+    input_type: "integers"
+    slo: 2000
+    default_value: "message not received"
+
+links:
+  link1:
+    app_name: "Application1"
```

Comment:
Let's make sure we're linking models with applications that have the same input type

Reply:
Done